jest.config.cjs

```diff
@@ -2,7 +2,10 @@
 module.exports = {
   transform: {
     "^.+\\.tsx?$": "ts-jest",
-    "\\.st\\.css?$": require.resolve("@stylable/jest"),
+    "\\.st\\.css?$": [
+      "@stylable/jest",
+      { stylable: { resolveNamespace: (ns, _srcPath) => `${ns}__Test` } },
```

Comment:
@mnapthine this ends up being: 
```js
  <div className="HelpText__Test__helpText">
```

Happy to alter if you feel it can be named better/differently.